hebrew/bar/Strawberries/desert-strawberries.md

```diff
@@ -0,0 +1,2 @@
+��� ���� ����, ��� �����!!
+����� �� �� �����?
```

Comment:
It looks like there is some encoding problem - your files are not displaying properly.